STEERING-COMMITTEE.md

```diff
@@ -100,19 +97,25 @@ first name):
 \* Open seats have been allocated to the organization per the committee rules,
   but have not been assigned to a member of the organization.
 
-### Allocation of seats
+### Composition
+
+As of 2020, the three organizations with the largest demonstrated 
+commitment to the project are Google, IBM/RedHat, and VMWare. For
+2021, a 5 person steering committee should be competent to
+execute its responsibilities, with three seats for Google, one seat
+for IBM/RedHat, and one seat for VMWare.
 
-Seats on the steering committee are allocated based upon contribution to the
-project by an organization. No final decision has been made on the exact
-formula.
+### Nomination of representatives
 
-As the project continues to grow, we expect to add additional seats to the
-committee, ensuring that those contributing to the project are properly
-represented.
+Seats on the steering committee are allocated based upon lifetime
+contributions to the project by an organization. The intention is
+to ensure that those organizations are committed to the ongoing
+maintenance burden that adding to core implies, and express their
```

Comment:
Does the intention to ensure organisations are committed to ongoing maintenance not seem a little at odds with a lifetime contribution criteria? If an organisation were no longer maintaining the project but did in the past, that seems like it'd be the opposite of demonstrating a commitment to ongoing maintenance. Can you see any argument for including at least an element of recent contribution, too?